BitmovinYoSpaceModule/Classes/BitmovinYospacePlayer.swift

```diff
@@ -101,19 +101,32 @@ public class BitmovinYospacePlayer: NSObject, Player {
     public var isOutputObscured: Bool { return player.isOutputObscured }
 
     @available(iOS 15.0, *)
-    public var sharePlay: BitmovinPlayer.SharePlayApi { return player.sharePlay }
+    public var sharePlay: SharePlayApi { return player.sharePlay }
+
+    public var _modules: _PlayerModulesApi { player._modules }
+
+    public var events: PlayerEventsApi {
+        BitLog.w(
+            """
+            Using `Player.events` is discouraged in combination with the Yospace Integration. \
+            Events will not contain Yospace related data. Use `Player.add(listener:)` instead.
+            """
+        )
+        return player.events
+    }
```

Comment:
We need to think about how to support the modern, Combine based, event listening with the Yospace integration. Currently this goes directly to the underlying player instance and doesn't filter any events, or emits e.g. Ad events.